src/esp/gfx/GenericDrawable.cpp

```diff
@@ -198,10 +198,8 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
       // fragment shader
       .setObjectId(
           static_cast<RenderCamera&>(camera).useDrawableIds() ? drawableId_
-          : ((((flags_ & Mn::Shaders::PhongGL::Flag::InstancedObjectId) ==
-               Mn::Shaders::PhongGL::Flag::InstancedObjectId) ||
-              ((flags_ & Mn::Shaders::PhongGL::Flag::ObjectIdTexture)) ==
-                  Mn::Shaders::PhongGL::Flag::ObjectIdTexture))
+          : (((flags_ >= Mn::Shaders::PhongGL::Flag::InstancedObjectId) ||
```

Comment:
Why do we compare flags with >=? Isn't flags basically a bitfield?

Reply:
that's how magnum compares multi-bit flags

Reply:
[Docs for reference](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html#aeb270daa9a8afb3a80e57ef2cb2f8314), `>=` is chosen because it looks close to a ⊇, a symbol for a superset.

Reply:
Ok so, what the heck do these lines mean? :) Is it testing whether `flags_` includes `InstanceObjectId`?

Reply:
> Is it testing whether `flags_` includes `InstanceObjectId`?

Yes. It's a shorthand for `(a & other) == other`, as the docs say. It does exactly what it did before -- checking if one set of bits is a superset of the other set -- just in a more concise way.